pkg/logs/input/channel/tailer.go

```diff
@@ -51,11 +53,7 @@ func (t *Tailer) run() {
 		origin := message.NewOrigin(t.source)
 		tags := origin.Tags()
 
-		if logline.Lambda != nil && logline.Lambda.FunctionName != "" {
-			origin.SetService(logline.Lambda.FunctionName)
-		} else {
-			origin.SetService("agent")
-		}
+		origin.SetService(computeServiceName(logline.Lambda, os.Getenv("DD_SERVICE")))
```

Comment:
Should we use a constant for this env var? And maybe add a comment noting this is only supported in the Serverless agent.

Reply:
✅